pkg/skaffold/build/tag/env_template.go

```diff
@@ -32,10 +32,21 @@ type envTemplateTagger struct {
 
 // NewEnvTemplateTagger creates a new envTemplateTagger
 func NewEnvTemplateTagger(t string) (Tagger, error) {
+	if strings.Contains(t, "{{.DIGEST}}") {
```

Comment:
This wouldn't work if it has a space or do we normalize it somewhere? e.g `{{ .DIGEST }}`

Reply:
This has been fixed